### ISSUE TITLE
training stuck at the beginning

### ISSUE THREAD

**Issue by winiel559** [ZFTurbo/Keras-RetinaNet-for-Open-Images-Challenge-2018#17]
Hello, I started training as README says and download the data from .
when running `retinanet_training_level_1/find_image_parameters.py`
I couldn't really start training.
The program stuck at the output `Labels length: 443` for more than two days.
I found the code stuck at line 332 of `retinanet_training_level_1/open_images_generator.py`
It is a while loop trying to get a batch of images to train, which only breaks when `len(self.class_index_array[random_class]) > 0:`
I print out `self.class_index_array`, it has 443 lists and every list is empty.

Is this normal or can you help me fix this? thank you.

This is my environment:
python 3.5.6
tensorflow-gpu 1.14.0
keras 2.2.5
keras-applications 1.0.8
keras-preprocessing 1.1.0
keras-resnet 0.1.0
keras-retinanet 0.4.1
opencv 3.4.2

if any more information is needed, please tell me.

Thanks for your help.

**Comment by ZFTurbo** [comment]
Looks like your dataset location is wrong. Check paths of input data.

**Comment by winiel559** [comment]
Tks, i've found the problem.
In find_image_parameters.py, the valid path is dataset_path+'validation_big'. But my dir_name is 'validation' instead.
And at line 44, 45     
```
if type == 'train':
        files = glob.glob(STORAGE_PATH_TRAIN + '*/*.jpg')
```
the `*/*` made the program find zero images.

Thanks for your help.
